torchvision/prototype/transforms/functional/_color.py

```diff
@@ -41,10 +53,40 @@ def adjust_contrast(inpt: features.InputTypeJIT, contrast_factor: float) -> feat
         return adjust_contrast_image_pil(inpt, contrast_factor=contrast_factor)
 
 
-adjust_sharpness_image_tensor = _FT.adjust_sharpness
+def adjust_sharpness_image_tensor(image: torch.Tensor, sharpness_factor: float) -> torch.Tensor:
+    if sharpness_factor < 0:
+        raise ValueError(f"sharpness_factor ({sharpness_factor}) is not non-negative.")
+
+    _FT._assert_image_tensor(image)
+
+    _FT._assert_channels(image, [1, 3])
```

Comment:
Since we are copy-pasting code, let's take a moment to rewrite on our side these 2 checks. If you see what they do internally, they are pretty bad generating multiple duplicate calls. Could you just rewrite these 2 assertions here?

Reply:
Sure.